runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseTest.java

```diff
@@ -464,13 +516,17 @@ private String locateGo() {
 		return prop;
 	}
 
-	private String locateRuntime() {
+	private static File locateRuntime() {
 		final ClassLoader loader = Thread.currentThread().getContextClassLoader();
 		final URL runtimeSrc = loader.getResource("Go");
```

Comment:
>I concluded by deduction

I was wondering how you deduced it. If I were writing this, I would have no idea that resource with that name is there, or that I should use it instead of manipulating the GO_RUNTIME_IMPORT_PATH string above.

>It wouldn't work otherwise

Why is the resource used instead of GO_RUNTIME_IMPORT_PATH? It seems to be redundant. Is the "Go" resource sometimes not the parent directory of GO_RUNTIME_IMPORT_PATH?

>I can not deny nor confirm that a Println or a debugger was involved in this.

Heh! Just to be clear, I'm not asking to be critical, I'm just asking because I had no idea it was there or what it was.